app/views/admin/logs/_form.slim

```diff
@@ -0,0 +1,22 @@
+= simple_form_for([:admin, @log], remote: true) do |f|
+  
+  = f.hidden_field(:project_step_id)
+  
+  = f.error_notification
+  
+  div.row  
+    div.date.col-lg-9
+      = f.input :date, as: :date
+      = check_box_tag(:step_completed_on_date)
+      span = t('log.step_completed')
```

Comment:
Important to always wrap checkboxes and radio buttons in a label tag so that you can click the label _or_ the checkbox.